reana_commons/utils.py

```diff
@@ -453,3 +454,36 @@ def build_unique_component_name(component_type, id=None):
         component_type=component_type,
         id=id or str(uuid.uuid4()),
     )
+
+
+def get_usage_percentage(usage: int, limit: int) -> str:
+    """Usage percentage."""
+    if limit == 0:
+        return ""
+    return "{:.0%}".format(usage / limit)
+
+
+def get_quota_resource_usage(
+    resource: Dict, human_readable_or_raw: str
+) -> Tuple[str, bool]:
+    """Return quota resource usage and health.
+
+    :param resource: Dict representing quota resource obtained from get_quota_usage()
+    :param human_readable_or_raw: One of ("human_readable", "raw")
+
+    :return: Tuple containing quota resource usage string and resource health.
+        i.e. ("1 MiB out of 10 MiB used (10%)", "healthy")
+
+    """
+    usage = resource.get("usage")
+    limit = resource.get("limit")
+    limit_str = ""
+    percentage = ""
+    health = None
```

Comment:
*suggestion:* In the function definition, you indicated that the return type is `Tuple[str, bool]` meaning that `health` is type `bool`. 

On the line above, `health` equals `None` which already forces `health` to be `Optional[bool]`. But, even more, `health` is not a bool, it is string ([code](https://github.com/reanahub/reana-db/blob/7b9b1a999860a9a18c475459929c249ba4276682/reana_db/models.py#L87)) meaning return type should be `Optional[str]`.

If `Optionals` sounds too complex to add, we could make the default value for `health` as `healthy`. But I am not sure what implications will be for the user output.